etc/opensc.conf.in

```diff
@@ -301,6 +301,69 @@ app default {
 		# md_guid_as_label: Use the GUID generated for the key as label in the PKCS#15 structure (Default: false - no label set)
 		# md_supports_container_key_gen: Card allows generating key pairs on the card (Default: false)
 		# md_supports_container_key_import: Card allows importing private keys (Default: false)
+		#
+		# Window title of the PIN pad dialog
```

Comment:
Not sure if this is the best place to store locale strings. It scales for two or three languages, but maintaining it for little bit more sounds like a lot of pain. We the `gettext()` could not be used?

Also this is growing the configuration file significantly from ~800 lines in 0.17.0 adding almost ~400 more. I understand that it is the simplest place to put them, but I would rather see it someplace else. The configuration is already in top 10 largest files in my `/etc` with `26K`.

Reply:
For po files, there are many tools that make it very simple to translate the strings even for non-technical people. Using these strings should be also pretty simple (though I didn't do that in C yet). In Fedora wiki, we have pretty nice guide how to do that [1]


Certainly, it is better to have languages separated from minidriver code, but from my POV, configuration file is not the place for it either.

Reply:
I looked at PO localization and didn't find any advantage over hardcoding the localization strings in the C-code other than avoiding a big switch statement. That didn't convince me to learn PO and to look deeper into how PO would be used on Windows (where localization is not derived from `LANG` and its friends.

Which brings me to how localization is done with this PR: All *default* language strings are hard-encoded in `ui/strings.c` in a big `switch` statement (currently English and German). From your comments I see that you mistakenly take `opensc.conf` as the place where these strings are stored. This is only the case if customized strings should be used (i.e. not the default localized strings from `ui/strings.c`). The strings that got added to `opensc.conf` are only specific to one single card, which has a builtin fingerprint sensor (hence, the default message for verifying the PIN doesn't apply).